
---
string message
int32 left
int32 right
